kbase-extension/static/kbase/config/data_source_config.json

```diff
@@ -290,21 +290,21 @@
                         "Genome"
                     ],
                     "displayName": "Example Genomes",
-                    "header": "Genomic sequence generally with attached functional annotations"
+                    "header": "Genomic sequence with attached functional annotations"
                 },
                 {
                     "name": [
                         "FBAModel"
                     ],
                     "displayName": "Example FBAModels",
-                    "header": "A metabolic model of an organism"
+                    "header": "Genome-scale metabolic models of organism"
```

Comment:
should be "models of organisms"

Reply:
@briehl - Which branch should use for the PR? I accidentally picked the default which was kbase:develop.